eventsProject/singleEventPage.php

```diff
@@ -7,32 +7,62 @@
 	wrongPage();
 }
 
-$event=getOneEvent($_REQUEST['eventID']); //should be $_REQUEST
+$eventID=($_REQUEST['eventID']);
+$event=getOneEvent($eventID);
 $categories=getCatsForThisEvent($_REQUEST['eventID']);
 $usersGoing=getUsersGoingToThisEvent($_REQUEST['eventID']);
 
 $creator=getWhoCreatedEvent($_REQUEST['eventID']);
 
 echoHeader($event['name'], null);
 
+$myEventVotes=$event['votes'];
 
-// <div class='centeredPictureText'>
-// <div class='row'>
-// 	<div class='left'>
-// 		$event[votes]
-// 	</div>
-// 	<div class='right'>
-// 		$event[name]
-// 	</div>
-// </div>
+?>
+<script src="/include/jquery.js"></script>
 
-//<div class='centeredPictureText'>$event[votes] $event[name]</div>
 
-echo "
+<script type='text/javascript'>
+
+function upVoteNumber($myEventVotes, $eventID){
+
+	var varEventVotes = "<?php echo $myEventVotes ?>";
+	var varEventID = "<?php echo $eventID ?>";
+
+	$.post('/ajax/upVoteAjax.php', {eventVotes:varEventVotes, eventID:varEventID},).done(function(data) {
+		console.log(data);
+		document.getElementById("votes").innerHTML = data;
```

Comment:
This works, but you should try to do an optimistic write. Rather than waiting for the ajax call to come back, update the number instantly. This will be more important once this is online. On your local machine the ajax call comes back right away, but it will take longer on the internet.